spacy_llm/pipeline/llm.py

```diff
@@ -37,6 +38,7 @@ class CacheConfigType(TypedDict):
             "batch_size": 64,
             "max_batches_in_mem": 4,
         },
+        "save_io": False,
```

Comment:
I personally think we should default this to True. Is memory consumption/storage space the only concern here?

Reply:
I think so. I'm on the fence about whether to make this opt-out.

Reply:
It will be opt-in for now (see [comment](https://github.com/explosion/spacy-llm/pull/127#issuecomment-1569706839))